### ISSUE TITLE
Lando Start -Pantheon pre-run scripting - file_put_contents(./composer.json): failed to open stream: Permission denied  

### ISSUE THREAD

**Issue by bsnodgrass** [lando/lando#936]
Similar issue previously identified and closed. Lando start ... could not create Drush or composer.json ... Permission Denied #493

Bug Report
==========

**Tell us about your setup**

lando-v3.0.0-beta.43 on 10.13.4 (17E199)

Site is Drupal 7 on Pantheon, not built using composer.

```yml
name: atmiprecast
recipe: pantheon
config:
  framework: drupal
  site: atmiprecast
  id: 7f95d4d3-3695-4883-b916-ed3b2c8204a2
```

```yml
$ cat pantheon.yml
---
api_version: 1

# PHP Version: 
# https://pantheon.io/docs/pantheon-yml#php-version
# Set site's PHP version to 7.0
php_version: 7.0

# Drush Version
# https://pantheon.io/docs/pantheon-yml/#drush-version
drush_version: 8

# Protected Web Paths
# https://pantheon.io/docs/pantheon-yml#protected-web-paths
protected_web_paths:
  - /CHANGELOG.txt
  - /README.txt

# Quicksilver Platform Integration Hooks
# https://pantheon.io/docs/pantheon-yml#quicksilver
workflows:
  clone_database:
    after:
      - type: webphp
        description: Enable development modules on dev
        script: private/scripts/enable_dev_modules/enable_dev_modules.php
```

Running lando start for the first time

```bash
$lando start [OK to Pantheon pre-run scripting]
Changed current directory to /var/www/.composer
                                                                                
  [ErrorException]                                                              
  file_put_contents(./composer.json): failed to open stream: Permission denied 

```

**To get around this I am running lando start multiple times. Once the app container is started the composer files can be written.**

I do have composer installed globally and per discussion Slack:

dustinleblanc [2:01 PM]
The default recipe does not, however, all PHP services are currently installing Prestissimo via Composer to try to be helpful.

Turns out that was a stupid way to add it
dustinleblanc [2:03 PM]
We're using CGR to install it globally in the container to speed up Composer stuff, We're probably going to move it into the PHP images themselves rather than at build time to speed up the builds

It would also avoid odd issues like this one with permissions

**Comment by bsnodgrass** [comment]
@dustinleblanc I think you had a handle on this yesterday.. I posted your comments from slack as a reminder

**Comment by yumyo** [comment]
@nterbogt thanks, it worked for me. However, I am not getting a:

**Comment by dustinleblanc** [comment]
Fixed in 0b21825c7a8836700753ed3bdf9618577e5adcbe
